projects/step-frontend/src/lib/modules/timeseries/time-series.service.ts

```diff
@@ -23,4 +24,8 @@ export class TimeSeriesService {
   rebuildTimeSeries(executionId: string): Observable<AsyncTaskStatusResource> {
     return this.http.post<AsyncTaskStatusResource>(`rest/time-series/rebuild`, { executionId: executionId });
   }
+
+  verifyOql(oql: string): Observable<OqlVerifyResponse> {
+    return this.http.get<OqlVerifyResponse>(`rest/time-series/oql-verify?oql=${oql}`);
```

Comment:
Is this not exposed over our openAPI generated client?

Reply:
it should be. currently it is not. how should I generate it?